tgui/packages/tgui/interfaces/Quickload_TGMC/NameInputModal.tsx

```diff
@@ -0,0 +1,57 @@
+import { useLocalState } from "../../backend";
+import { Button, Flex, Modal, TextArea } from "../../components";
+import { NameInputModalData } from "./Types";
+
```

Comment:
why does this file exist you dont use it anywhere

ditto for the DataInputModal type

Reply:
Yeah I suspected that but wasn't certain.again I'm just a monkey with a rock, I don't really know tgui.